client/components/tinymce/plugins/contact-form/dialog/index.jsx

```diff
@@ -1,13 +1,98 @@
 /**
  * External dependencies
  */
+import React, { PropTypes } from 'react';
 import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
-import Dialog from './dialog';
+import Dialog from 'components/dialog';
+import FormButton from 'components/forms/form-button';
+import FormSettings from './settings';
+import Navigation from './navigation';
+import FieldList from './field-list';
+
+const ContactFormDialog = React.createClass( {
+	displayName: 'ContactFormDialog',
+
+	propTypes: {
+		activeTab: PropTypes.oneOf( ['fields', 'settings'] ).isRequired,
```

Comment:
Arrays should have spaces between square brackets ([reference](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#spacing))